Sources/Nimble/Matchers/BeAKindOf.swift

```diff
@@ -16,15 +16,21 @@ public func beAKindOf(expectedClass: AnyClass) -> NonNilMatcherFunc<NSObject> {
     return NonNilMatcherFunc { actualExpression, failureMessage in
         let instance = try actualExpression.evaluate()
         if let validInstance = instance {
-            failureMessage.actualValue = "<\(NSStringFromClass(validInstance.dynamicType)) instance>"
+            failureMessage.actualValue = "<\(classAsString(validInstance.dynamicType)) instance>"
         } else {
             failureMessage.actualValue = "<nil>"
         }
-        failureMessage.postfixMessage = "be a kind of \(NSStringFromClass(expectedClass))"
+        failureMessage.postfixMessage = "be a kind of \(classAsString(expectedClass))"
+
+#if !os(Linux)
         return instance != nil && instance!.isKindOfClass(expectedClass)
+#else
+        fatalError("Unimplemented")
```

Comment:
If we can't use `isKindOfClass` for linux, I don't think we need this matcher at all. It's really only for objective-c objects anyway.